app/containers/Menu/MenuItems.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import List from '@material-ui/core/List';
+import Divider from '@material-ui/core/Divider';
+import ListItem from '@material-ui/core/ListItem';
+import { FoodName,
+  IngredientsList,
+  FoodTextInfoWrapper,
+  FoodDescriptionWrapper,
+  FoodPicture,
+  PriceWrapper,
+  FoodPictureWrapper } from './styles';
+
+const MenuItems = ({ menu, section, isLoading }) => (
+  <List component="nav">
+    {!isLoading && (menu[section].dishes).map((item, i) =>
+      (
+        <div key={item.name} >
+          <ListItem button>
+            <FoodDescriptionWrapper>
+              <FoodTextInfoWrapper>
+                <FoodName>
+                  {item.name}
+                </FoodName>
+                <IngredientsList>
+                  {item.ingredients.reduce((ingredient, curr) => `${curr}, ${ingredient}`)}
+                </IngredientsList>
+                <PriceWrapper>
+                  {`R$ ${item.price},00`}
```

Comment:
It won't work with decimal prices, like 1.99. I created I function called convertToReal at /utils. Please, check the branch bill-component.